typescript-timers/interval.ts

```diff
@@ -0,0 +1,11 @@
+const $countdown = document.querySelector('.countdown-display');
+if (!$countdown) throw new Error('$countdown query failed');
+let timer: number = 4;
+setInterval(() => {
+  if (timer === 0) {
+    $countdown.textContent = '~Earth Beeeelooowww Us~';
```

Comment:
We need to make sure that the setInterval timer is cancelled in this code block. Otherwise it will run infinitely. This wouldn't be a performance issue on a site like this but in the real world should be done.